libmscore/read206.cpp

```diff
@@ -1933,6 +1935,17 @@ static void readMeasure(Measure* m, int staffIdx, XmlReader& e)
                   segment = m->getSegment(SegmentType::ChordRest, e.tick());
                   segment->add(el);
                   }
+            else if (tag == "Tempo") {
+                  Element* el = Element::name2Element(tag, score);
+                  // hack - needed because tick tags are unreliable in 1.3 scores
+                  // for symbols attached to anything but a measure
+                  el->setTrack(e.track());
+                  el->read(e);
+                  TextBase* tt = static_cast<TextBase*>(el);
+                  tt->setXmlText(tt->xmlText().replace("<sym>unicode", "<sym>met"));
```

Comment:
Seems this is the cause for the Travis failure, not sure whether it is the tests than need adjusting?